configure.py

```diff
@@ -439,6 +439,12 @@
     dest='v8_options',
     help='v8 options to pass, see `node --v8-options` for examples.')
 
+parser.add_option('--with-ossfuzz',
+    action='store_true',
+    default=False,
```

Comment:
```suggestion
```
(not that it really hurts to be explicit but other store_true options aren't either)

Reply:
Agreed.